docs/tutorial/kymotracking.rst

```diff
@@ -316,14 +316,20 @@ Extracting summed intensities
 -----------------------------
 
 Sometimes, it can be desirable to extract pixel intensities in a region around our kymograph track. We can quite easily
-extract these using the method :func:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack.sample_from_image`. For instance,
-if we want to sum the pixels in a 11 pixel area around the longest kymograph track, we can invoke::
+extract these using the method :func:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack.sample_from_image`.
+
+Prior to version 1.1.0 the method :meth:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack.sample_from_image` had a bug that assumed the origin of a pixel to be at the edge rather than the center of the pixel.
+Consequently, the sampled window could frequently be off by one pixel.
+The old (incorrect) behavior is maintained until the next major release (2.0) to ensure backward compatibility.
+It is recommended to include the argument `correct_origin=True` which results in using the correct origin.
```

Comment:
```suggestion
.. warning::
    Prior to version `1.1.0` the method :meth:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack.sample_from_image` had a bug that assumed the 
    origin of a pixel to be at the edge rather than the center of the pixel.
    Consequently, the sampled window could frequently be off by one pixel.
    The old (incorrect) behavior is maintained until the next major release (version `2.0`) to ensure backward 
    compatibility. It is recommended to include the argument `correct_origin=True` which results in using the correct origin.
```